src/app/home/upcoming-events/upcoming-events.component.ts

```diff
@@ -12,10 +12,16 @@ export class UpcomingEventsComponent implements OnInit {
   @Input() upcomingEvents: any[] = [];
   swiperModule = [IonicSlides];
 
+  titleColor: string = '';
+  titleText: string = '';
+
   constructor() {}
   // TODO: redirect to  details page of the selected event
   // TODO: add events interface
   // TODO: attach like functionality to like icon, _id is passed
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.titleColor = 'orange';
+    this.titleText = 'Предстоящи събития';
```

Comment:
Same here, isn't it better the title text to be passed from outside? This shouldn't be changed as much as the color, but just in case if we reuse the component in other parts of the app with other color and title